plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialBlockType.java

```diff
@@ -99,9 +102,13 @@ public Campfire getCampfire() {
         return (Campfire) material.getModernData();
     }
 
-    /*public PointedDripstone getDripstone() { // TODO: 1.17
+    public PointedDripstone getDripstone() {

```

Comment:
Did you think about what you were doing or did you just intentionally break this

Reply:
That was Aya, I just didn't see this when I was porting over changes